packages/components/src/dropdown-menu/index.tsx

```diff
@@ -154,13 +84,14 @@ function DropdownMenu( dropdownMenuProps: DropdownMenuProps ) {
 	const mergedPopoverProps = mergeProps(
 		{
 			className: 'components-dropdown-menu__popover',
+			variant,
 		},
 		popoverProps
 	);
 
 	return (
 		<Dropdown
-			className={ classnames( 'components-dropdown-menu', className ) }
+			className={ className }
```

Comment:
The context system automatically generates a `components-*` `classname` based on which namespace is passed in the `useContextSystem` hook — therefore making the `components-dropdown-menu` hardcoded classname unnecessary, since duplicated.